models/user.py

```diff
@@ -177,7 +177,7 @@ def generate_fake(count=100, **kwargs):
                 db.session.rollback()
 
     def __repr__(self):
-        return '<User \'%s\'>' % self.full_name()
```

Comment:
Using escaped single quotes (\') can be helpful when you specifically want to include single quotes around the formatted value. However, in most cases, you can achieve the same result without escaping the single quotes